bin/tx-prover/src/commands/mod.rs

```diff
@@ -56,6 +60,8 @@ impl Default for ProxyConfig {
             max_req_per_sec: 5,
             available_workers_polling_time_ms: 20,
             health_check_interval_secs: 1,
+            prometheus_host: "0.0.0.0".into(),
```

Comment:
Should we restrict this to localhost by default maybe?

Reply:
I'm changing it to `127.0.0.1` 👌🏼